src/irmin/conf.ml

```diff
@@ -43,7 +43,7 @@ let bool =
   ( (fun s ->
```

Comment:
The new cases for `indicate-multiline-delimeters` don't seem to apply to the opening paren (unlike the previous cases). Presumably that's for space reasons? It looks a bit odd to me, but I can see why one might prefer that to the consistent choice.

Reply:
We chose to have an exception for `fun` and `function` to never have a space after the opening parenthesis in this case in this context.